cli/package.json

```diff
@@ -110,5 +110,15 @@
   "engines": {
     "node": ">=12.0.0"
   },
-  "types": "types"
+  "types": "types",
+  "exports": {
+    ".": {
+      "import": "./index.mjs",
+      "require": "./index.js"
+    },
+    "./package.json": {
```

Comment:
Q: what does this do?

Reply:
https://nodejs.org/api/packages.html#package-entry-points allows package.json to be accessible. See "Package Entry Points"